applications/GeoMechanicsApplication/custom_constitutive/elastic_3D.h

```diff
@@ -0,0 +1,31 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Gennady Markelov
+//
+
+#pragma once
+
+#include "constitutive_law_dimension.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) Elastic3D : public ConstitutiveLawDimension
```

Comment:
I'm doubting a bit about the name of the class, the plane strain counterpart is just called PlaneStrain, but I'm not sure if ThreeDimensional would be descriptive enough.

Reply:
For me it is.

Reply:
What do @rfaasse and @WPK4FEM  think about `ElasticIsotropic3D` and to change `ConstitutiveLawDimension` to `ConstitutiveLawType`? For example, there is `GeoLinearElasticPlaneStress2DLaw` which fits this structure perfectly using `PlaneStress` like `PlaneStrain`. Another candidate is `ElasticIsotropicK03DLaw`. Thank you.